build.rs

```diff
@@ -3,12 +3,21 @@
 
 use std::env;
 
+#[cfg(feature = "kv_unstable")]
+#[path = "src/kv/value/internal/cast/into_primitive.rs"]
+mod into_primitive;
+
 fn main() {
     let target = env::var("TARGET").unwrap();
 
     if !target.starts_with("thumbv6") {
         println!("cargo:rustc-cfg=atomic_cas");
     }
 
+    #[cfg(feature = "kv_unstable")]
+    into_primitive::generate();
+
+    println!("cargo:rustc-cfg=src_build");
```

Comment:
I’m using this so the `into_primitive` module can tell if it’s in the build script or not. Is there a cfg that exists already for this?